Userland/Libraries/LibC/stdlib.h

```diff
@@ -10,14 +10,14 @@
 #include <sys/cdefs.h>
 #include <sys/types.h>
 
-__attribute__((warn_unused_result)) int __generate_unique_filename(char* pattern);
-
 __BEGIN_DECLS
 
 #define EXIT_SUCCESS 0
 #define EXIT_FAILURE 1
 #define MB_CUR_MAX 4
 
+__attribute__((warn_unused_result)) int __generate_unique_filename(char* pattern);
+
```

Comment:
Why is this change needed? This function isn't even defined anywhere, as it's been replaced by the un-uglified `generate_unique_filename` template, which is only used in `stdlib.cpp`. A better change would be to remove this declaration completely.

Reply:
Not related, I can make a separated PR about this. I found it when I needed to move `_fork` to stdlib, in a patch not pushed yet.